controllers/managedocs_controller.go

```diff
@@ -1408,6 +1420,37 @@ func (r *ManagedOCSReconciler) removeOLMComponents() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) removeStorageSystem() error {
+	storageSystemList := odfv1a1.StorageSystemList{}
+	if err := r.list(&storageSystemList); err != nil {
+		return fmt.Errorf("unable to list storageSystem resource: %v", err)
+	}
+	for i := range storageSystemList.Items {
+		storageSystem := storageSystemList.Items[i]
+		if err := r.delete(&storageSystem); err != nil {
+			return fmt.Errorf("unable to delete storageSystem: %v", err)
+		}
+	}
+	return nil
+}
```

Comment:
This is a util function that is only used once in a specific location and is very short. 
I see no need for a function in this case. Can we please inline this, with a comment at the start saying:
"Deleting all storage systems from the namespace"?